src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylePairs = sourceString
+    .split(';')
+    .filter((pair) => pair.trim() !== '');
+
+  const stylesObject = {};
+
+  stylePairs.forEach((pair) => {
+    const [key, value] = pair.split(':').map((part) => part.trim());
```

Comment:
It's good to use `.map` to trim both the key and value, but be aware that this code does not handle the case where there might be multiple colons within a property value. While not required to handle this case, it is something to consider if the function needs to be more robust in the future.